samples/KeyVault/disallow-keyvault-access-policies/azurepolicy.parameters.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file should have the parameters from the azurepolicy.json file

`      "aadObjectId": {
        "type": "String",
        "metadata": {
          "displayName": "AAD Object ID",
          "description": "The objectId of the AAD user, group, or service principal to check."
        }
      }`

Reply:
Done!